rules/mech-building/overview.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: template.njk
+title: Mech Building Overview
+permalink: /rules/build-overview/
+tags: rules_page
+---
+
+## Overview
+
+Mech building is at the heart of the Mechtronica RPG system and as such has some of the most in depth mechanics. To start we'll introduce the key concepts here before you dive into the details.
+
+The mechtronica system is highly modular. It's not like picking a car model and choosing trims and packages, you're building this from the ground up. You will pick a frame which is where all the other parts for your mech will go, but the main thing it determines are how fast your mech is and how many parts it can fit.
+
+Nearly all parts use {% keyword "Resources" glossary %}. Resources (and the related {% keyword "Power Type" glossary %}) are the main constraints on your mech build, you have to make sure you have enough resources for all your {% keyword "Infrastructure Components" glossary %} to operate and enough left over to use your {% keyword "Attachment Components" glossary %} when you need to.
+
+Most infrastructure components are grouped into "Subsystems". These groups of components interact with each other closely and set constraints on each other. There are really just two such subsystems, the "Control Subsystem" and the "Power Subsystem". The control subsystem components are related by software type and CPU power. The power subsystem components are related by fuel type.
```

Comment:
Typically I've been trying to use the term "Mainframe" to refer to the actual component, and only use CPU/Processing to refer to the resource. Probably best to change CPU here to "Mainframe", no?

Reply:
CPU there does refer to the resource. I could say "CPU resource" instead to clarify. The CPU provided by the mainframe is used primarily by the modules. I might also reference the modules being placed in the cabin as a relationship there, but maybe I don't even need to mention any of that stuff at all. 
I probably should mention the Frame/Plating group which is basically a subsystem, do we have a name for that?

Reply:
I grouped plating into the same section as Frames on the Components page. Perhaps "Frame Subsystem"? Or maybe just "Superstructure"?

Reply:
Fixed